docs/sources/write/style-guide/word-list/index.md

```diff
@@ -19,10 +19,31 @@ In most cases, you can refer to the [word list in the Google developer documenta
 This page is a work in progress.
 {{< /admonition >}}
 
-<!-- vale off -->
+<!-- vale Grafana.Headings = NO -->
+<!-- vale Grafana.Spelling = NO -->
+<!-- vale Grafana.WordList = NO -->
 
-<!--
 ## A
+
+### alert rule
+
+Grafana Alerting uses the term _alert rule_ to describe the Grafana feature that includes both [Grafana-managed alert rules](https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/alert-rule-types/#grafana-managed-alert-rules) and [Data source-managed alert rules](https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/alert-rule-types/#data-source-managed-alert-rules).
+For more information, refer to [Alert rules](https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/).
+
+{{< admonition type="caution" >}}
+You mustn't confuse this with an [_alerting rule_ ](#alerting-rule).
```

Comment:
```suggestion
Don't confuse this with an [_alerting rule_ ](#alerting-rule).
```

Would it be better to go with a more well known phrase? Mustn't isn't commonly used these days in American English.

Reply:
Yeah that works for me, I always default to https://www.ietf.org/rfc/rfc2119.txt words but when contracted _don't_ makes as much sense and for me evokes the same meaning.